charts/GPP-publicatiebank/README.md

```diff
@@ -0,0 +1,101 @@
+# gpp-publicatiebank
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
```

Comment:
where does this AppVersion come from?

Reply:
They come from the Chart.yaml (which we have added manually). I think it was the latest stable tag when we deployed it last :thinking: